### ISSUE TITLE
Some episodes don't play in high quality

### ISSUE THREAD

**Issue by danielr0206** [mukul500/AnimeXStream#152]
So here's the deal, I started using this app 'cause of the fast loading of episodes and the ad-free interface. Then, all of a sudden, I'm noticing that some episodes don't play/stream in 1080p, while others do. For example One Piece EP567 plays at 360p while EP568 plays at 720p!(which is OK). And also, Noragami EP1 only plays at 360p while EP2 maxes-out at 480p! Please fix this issue by adding more high quality streams!! Great work by the wayy!!

**Comment by harshal-99** [comment]
Its goanime's fault that they don't have high quality videos, maintainer will have to switch to different provider to solve this problem and I don't think it's that easy.

**Comment by danielr0206** [comment]
Yes so there is this site called twist.moe where you can watch anime in 1080p, I would highly recommend that the maintainer would use this site along with gogoanime for the streams.

**Comment by danielr0206** [comment]
Can you use 9anime.app for streams
